docs/RouterConfiguration.md

```diff
@@ -101,18 +101,19 @@ return by the search engine before returning the results to client. There is als
+| `filterItinerariesWithSameFirstOrLastTrip`           | If more than one itinerary begins or ends with same trip, filter out one of those itineraries so that only one remains. Trips are considered equal if they have same id and same service day. Non-transit legs are skipped during comparison. Before filtering, trips are sorted by their generalized cost. Algorithm loops through list from top to bottom. If itinerary matches from any other itinerary from above, it is removed from list.                                                                                                                                                                                                                     | boolean         | `false`                                                          |
+| `accessibilityScore`                                 | A experimental feature contributed by IBI which adds an sandbox accessibility "score" between 0 and 1 for each leg and itinerary. This can be used by by frontend developers to implement a simple traffic light UI.                                                                                                                                                                                                                                                                                                                                                                                                                                                | boolean         | `false`                                                          | 
+| `removeTimeshiftedItinerariesWithSameRoutesAndStops` | Set to true if you want only to see one result per group which goes through the same stops and route, in effect removing the same itineraries but at different time. Set to true if you want to see a greater variety of itineraries rather than the "best" ones.                                                                                                                                                                                                                                                                                                                                                                                                   | boolean         | `false`                                                          | 
```

Comment:
```suggestion
| `removeTimeshiftedItinerariesWithSameRoutesAndStops` | Set to true if you want to list only the first itinerary  which goes through the same stops and routes. Itineraries visiting the same set of stops and riding the exact same routes, departing later are removed from the result.                                                                                                                                                                                                                                                                                                                                                                                                   | boolean         | `false`                                                          | 
```
This do not lead to "better variety", only fewer itineraries in the result, so I removed the last sentence.

Reply:
I like `removeItinerariesWithSameRoutesAndStops` better than `removeTimeshiftedItinerariesWithSameRoutesAndStops`. In OTP we use the phrase *time-shifted* for something that is moved in time, these itineraries are separate entries, and the times in them may not be equally shifted.

Reply:
I've renamed it.